### ISSUE TITLE
Errors Building Mango Viz

### ISSUE THREAD

**Issue by Georgehe4** [bigdatagenomics/mango#421]
When running make develop 
for mango-viz, I run into the following error: https://gist.github.com/Georgehe4/91bd849caac046b65a81b10b5cef9d90

I suspect this is due to an out-of-date node installation.

**Comment by Georgehe4** [comment]
Suspicions were correct. After updating node to a more recent version than what was packaged/installed by the default package manager, the command ran successfully.
[specifically, I updated to node v8.11.4 and npm 5.6.0]

**Comment by Georgehe4** [comment]
Reopening -- @akmorrow13, do you think it's worth documenting the minimum node/npm versions required to build?

**Comment by akmorrow13** [comment]
Yes, this should be documented. Do you know the versions you ran that were failing?

**Comment by Georgehe4** [comment]
node -v v0.10.29
npm -v 1.4.21

We probably should just pin the requirements to a more recent version of npm regardless.
